setup.cfg

```diff
@@ -57,4 +62,21 @@ line_length = 127
 lines_between_types = 1
 lines_after_imports = 2
 known_first_part = build
-known_third_party = nox,pallets_sphinx_themes,pep517,pytest,setuptools,toml
+known_third_party = pallets_sphinx_themes,pep517,pytest,setuptools,toml
+
+[coverage:paths]
+source =
+    src
+    .tox/*/lib/python*/site-packages
+    .tox/pypy*/site-packages
+    .tox\*\Lib\site-packages\
+    */src
+    *\src
+
+[coverage:run]
+branch = true
+parallel = true
+dynamic_context = test_function
+
+[coverage:html]
+show_contexts = true
```

Comment:
This should be in a separate PR, or at least commit.

Reply:
I guess the merge coverage part (that we did not do until now) can be a follow-up PR.